encoder/vaapiencoder_vp9.cpp

```diff
@@ -257,10 +257,10 @@ bool VaapiEncoderVP9::fill(VAEncPictureParameterBufferVP9* picParam,
         }
     }
 
-    picParam->frame_width_src = width();
-    picParam->frame_height_src = height();
-    picParam->frame_width_dst = width();
-    picParam->frame_height_dst = height();
+    picParam->frame_width_src = ALIGN8(width());
```

Comment:
Hi @chivakker 
I am little confuse here. What if the source is not 8 pixel align? Will this introduce artificial if the video src is not 8 pixel aligned?